.github/workflows/test.yml

```diff
@@ -2,9 +2,9 @@ name: Tests
 
 on:
   push:
-    branches: [ master ]
+    branches:
+      -main
   pull_request:
-    branches: [ master ]
```

Comment:
Why do we remove this? Doesn't that disable running workflows on PR on `main`?

Otherwise LGTM

Reply:
No value means it runs with any pull request - otherwise this workflow would not be running right now.